topics/single-cell/tutorials/scatac-standard-processing-snapatac2/tutorial.md

```diff
@@ -0,0 +1,940 @@
+>    - {% icon param-file %} *"Replace information file"*: `replace_file` 
+>    - *"Which column should be replaced?*: `Column: 1`
+> 7. Rename the generated file to `Cell type annotation` and {% icon galaxy-eye %} inspect the file to check if the replacement was successful
+>    - check if the datatype is set to `tabular`
+>    - you may need to change the datatype manually
+> 
+>      {% snippet faqs/galaxy/datasets_change_datatype.md datatype="tabular"%}
+> 
+> 8. {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.10.3+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC gene_matrix magic UMAP`
+>    - *"Function to manipulate the object"*: `Add new annotation(s) for observations or variables`
+>        - *"What to annotate"*: `Observations (obs)`
+>        - {% icon param-file %} *"Table with new annotations"*: `Cell type annotation`(output of **Replace column** {% icon tool %})
+> 9. Rename the generated file to `Anndata 5k PBMC gene_matrix magic cell_type` or add the tag {% icon galaxy-tags %} `cell_type` to the dataset
+>
+> 10. {% tool [Plot with Scanpy](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_plot/scanpy_plot/1.9.6+galaxy2) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC gene_matrix magic cell_type` (output of **Manipulate AnnData** {% icon tool %})
+>    - *"Method used for plotting"*: `Embeddings: Scatter plot in UMAP basis, using 'pl.umap'`
+>        - *"Keys for annotations of observations/cells or variables/genes"*: `cell_type`
+>        - {% icon param-toggle %} *"Show edges?"*: `No`
+>        - In *"Plot attributes"*
+>           - *"Location of legend"*: `on data`
+>           - {% icon param-toggle %} *"Draw a frame around the scatter plot?"*: `Yes`
+> 11. {% icon galaxy-eye %} Inspect the `.png` output
+>
+>  ![UMAP annotated cell types]({% link topics/single-cell/images/scatac-standard-snapatac2/umap_cell-types.png %})
+>
+>    > <question-title></question-title>
+>    > Are clusters with the same assigned cell type located close to each other?
+>    > > <solution-title></solution-title>
+>    > >
+>    > > Yes. B-cells are far away from NK and T cells. Only the myeloid lineage of monocytes and dendritic cells are located close to each other. This might be due to the common progenitor cell lineage and thus a similar chromatin profile.  
+>    > {: .solution}
+>    >
+>    {: .question}
+>
+{: .hands_on}
+
+
+# Conclusion
+{% icon congratulations %} Well done, you’ve made it to the end! You might want to consult your results with this [control history](https://singlecell.usegalaxy.eu/u/timonschlegel/h/test-of-5k-pbmc-tutorial-workflow), or check out the [full workflow](https://singlecell.usegalaxy.eu/u/timonschlegel/w/2combined-snapatac2) for this tutorial.
```

Comment:
links to the control history and the workflow are invalid. Please check.

Reply:
strange, because they do work for me. I have updated my visable name a few days ago, maybe that is causing the issues? But since I updated the links it works for me. 

But I have to change the control history anyway before publishing, since I don't have a history that contains the entire process, because make_fragment_file isn't updated yet. And maybe I will also change the workflow, so that each step is listed individually? And I could add the make_bam-file steps as well